cmd/termui/provider/logProvider.go

```diff
@@ -0,0 +1,91 @@
+package provider
+
+import (
+	"encoding/hex"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/logger"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/gorilla/websocket"
+)
+
+var formatter = logger.PlainFormatter{}
+var webSocket *websocket.Conn
+
+// InitLogHandler will open the websocket and set the log level
+func InitLogHandler(nodeURL string, logLevel string) error {
+	var err error
+	webSocket, err = openWebSocket(nodeURL, fmt.Sprintf("*:%s", logLevel))
```

Comment:
This will generate a lot of log lines being displayed from the api package that are normal for this kind of operation.
```GO
DEBUG[2019-11-27 08:08:55] gin server                               message=[GIN] 2019/11/27 - 08:08:55 | 200 |     284.764µs |       127.0.0.1 | GET      /node/status
```
Might think of a solution to manually set api:INFO in a clever way.

Reply:
introduced a check for messages which contain '/node/status' for now

Reply:
ok